package.json

```
@@ -1,8 +1,8 @@
 {
-  "name": "javascript-app-template",
+  "name": "Harry Potter App",
```

This line throws an error which is why the tests are not running through on GitHub. I searched for the error message and this is what I found: https://webstoked.com/fix-sorry-name-can-only-contain-url-friendly-characters-npm/
Apparently the name needs to be url-friendly, so no spaces. Try to change this and let's see if that fixes the error!
```suggestion
  "name": "Harry-Potter-App",
```